test-files/cmd/glacial/PoiPolygonConflateCombinedTest.sh

```diff
@@ -18,6 +19,11 @@ hoot conflate -C UnifyingAlgorithm.conf \
   -D address.scorer.enable.caching=true \
   $inputDir/PoiPolygon1.osm \
   $inputDir/PoiPolygon2.osm \
-  $outputDir/output1.osm
+  $outputDir/output.osm
+hoot diff -C Testing.conf $outputDir/output.osm $checkDir/output.osm || diff $outputDir/output.osm $checkDir/output.osm
 
-hoot diff -C Testing.conf $outputDir/output1.osm $checkDir/output1.osm || diff $outputDir/output1.osm $checkDir/output1.osm
+if [ -f "test-output/test-validation-enabled" ]; then
+  hoot validate $LOG_LEVEL $CONFIG $outputDir/output.osm \
+    --report-output $outputDir/output-validation-report --output $outputDir/output-validated.osm
+  diff $checkDir/output-validation-report $outputDir/output-validation-report
+fi
```

Comment:
This seems to be replicated in a lot of places, we ought to centralize it into a function in a script and call that in all of the scripts that need it.

Reply:
good idea. will do.